src/index.html

```diff
@@ -11,6 +11,27 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-form search-form_big" data-qa="big">
+      <label class="search-form__label search-form__label_big">
+        <input
+            name="search-input__big"
+            data-qa="keypress"
+            class="search-form__input search-form__input_big"
+            type="text"
+            placeholder="Try &quot;Los Angeles&quot;"
+          >
```

Comment:
```suggestion
        <input
          name="search-input__big"
          data-qa="keypress"
          class="search-form__input search-form__input_big"
          type="text"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```
indentation should be 2 spaces
Pay attention to it next time